### PR TITLE
feat(Pass plan): sub components

### PR DESCRIPTION
**Pull request by FMorrison87**
The pass plan component has three identified potential list items that can appear once the app reaches the ‘pass’ state:

- `MnemonicListItem` component: A checkbox with a listed mnemonic, this row can receive a watcher based on identifying mnemonics as being watched in the ‘watcher’ panel. These mnemonics are auto-generated using mock data.  Each mnemonic has a pop-up triggered from the mnemonic name. (The internals of this are not hooked up yet)

- `SelectMenuListItem`: A list item with two stacked pieces with select menus identifying if ‘XYZ’ is selected, and if we are receiving telemetry. These will retain these exact values, and this list item should probably only appear once as we don’t have any indication of what ‘XYZ’ should be.

- `ExecutableListItem`: A list item that contains a rux-tree that houses an executable with a progress bar and a play/pause button. Expanding the tree will show a sub-list of mnemonics that exactly mimics the parent level mnemonic item listed above. These are rendered using the `MnemonicListItem` component. These executable tree items will be linked to the commands found in the commands.json file, and adding a command from the search interface at the bottom (which becomes usable in the pass state) creates this exact list item type.

Keep in mind these sub-components are not meant to be feature complete. The functionality of them still needs to be plugged in.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const options = {
 };
 
 const contact: Contact = generateContact(0, {
-  desiredSubsystems: ["Altitude", "Payload", "Power", "Propulsion", "Thermal"],
+  desiredSubsystems: ["Attitude", "Payload", "Power", "Propulsion", "Thermal"],
```
Comment (Paterma) [inline src/App.tsx:21]: So it _IS_ Attitude! I definitely thought this was a typo in the wireframes. Love a sassy subsystem

File: src/Command/Components/Watcher/MnemonicPopUp.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const MnemonicPopUp = ({ triggerValue, data }: PropTypes) => {
             rel="noreferrer"
             style={styles.linkStyles}
           >
-            Altitude
+            Attitude
```
Comment (Paterma) [inline src/Command/Components/Watcher/MnemonicPopUp.tsx:41]: This popup link takes you to the old "Investigate" app, we can update this to go to our new one. Not necessary in this PR, just making note of it. :)

**Review by Paterma — APPROVED** [review]
Excellent work!

File: src/Command/Components/PassPlan/PassList/SelectMenuListItem/SelectMenuListItem.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.pass_select-wrapper {
+    display: flex;
+    gap: 5%;
+}
+
+/* .pass_number-wrapper {
```
Comment (Paterma) [inline src/Command/Components/PassPlan/PassList/SelectMenuListItem/SelectMenuListItem.css:6]: Remove?

File: src/App.tsx (hunk 1 above)
Comment (NicDominguez) [reply src/App.tsx:21]: Looks like you're right  but I wouldn't be surprised if that was a typo in the wireframes. 
https://www.geeksforgeeks.org/what-is-satellite-subsystem/# 
^ uses both attitude and altitude but this is how it's described. "Detection of the satellite attitude: first you have to identify at which particular height the satellite is a" ...I could see a case made for either one.

**Pull request merged**